test/sendEmail.test.js

```diff
@@ -1,5 +1,10 @@
 require('dotenv').config();
 const { createTestAccount } = require('nodemailer');
+const sender = require('../src/mailing/sender');
+
+afterEach(() => {
+  jest.restoreAllMocks();
+});
```

Comment:
See other comments here; we should not need to mock anything in this test file.

Reply:
deleted the block with mock